opentech/static_src/src/app/webpack.base.config.js

```diff
@@ -1,83 +1,91 @@
-                loader: 'babel-loader',
-                include: [path.resolve(__dirname, './src')],
-                query: {
-                    presets: ['@babel/preset-react', '@babel/preset-env'],
-                    plugins: [
-                        'react-hot-loader/babel',
-                        '@babel/plugin-proposal-class-properties'
-                    ]
-                },
-            },
-            {
-                test: /\.js$/,
-                exclude: /node_modules/,
-                include: [path.resolve(__dirname, './src')],
-                loader: 'eslint-loader',
-                options: {
-                    configFile: path.resolve(__dirname, './.eslintrc'),
+        entry: {
+            submissionsByRound: COMMON_ENTRY.concat(['./src/submissionsByRoundIndex']),
+            submissionsByStatus: COMMON_ENTRY.concat(['./src/submissionsByStatusIndex']),
+        },
+
+        output: {
+            filename: '[name]-[hash].js'
+        },
+
+        plugins: [],
+
+        module: {
+            rules: [
+                {
+                    test: /\.jsx?$/,
+                    loader: 'babel-loader',
+                    include: [path.resolve(__dirname, './src')],
+                    query: {
+                        presets: ['@babel/preset-react', '@babel/preset-env'],
+                        plugins: [
+                            'react-hot-loader/babel',
+                            '@babel/plugin-proposal-class-properties'
+                        ]
+                    },
                 },
-            },
-            {
-                test: /\.scss$/,
-                use: [{
-                    loader: 'style-loader'
-                }, {
-                    loader: 'css-loader',
-                    options: {
-                        sourceMap: true
-                    }
-                }, {
-                    loader: 'sass-loader',
+                {
+                    test: /\.js$/,
+                    exclude: /node_modules/,
+                    include: [path.resolve(__dirname, './src')],
+                    loader: 'eslint-loader',
                     options: {
-                        sourceMap: true,
-                        data: '@import "main.scss";',
-                        includePaths: [
-                            path.join(__dirname, 'src')
-                        ]
-                    }
-                }]
-            },
-            {
-                test: /\.svg$/,
-                use: ['@svgr/webpack']
-            }
-        ]
-    },
+                        configFile: path.resolve(__dirname, './.eslintrc'),
+                    },
+                },
+                {
+                    test: /\.scss$/,
+                    use: [{
+                        loader: 'style-loader'
+                    }, {
+                        loader: 'css-loader',
+                        options: {
+                            sourceMap: !isProduction
```

Comment:
I don't think webpack adds any source maps by default in the production mode. Did it?

Reply:
it did because it is the css-loader not webpack adding the sourcemaps. Compare the *.scss sizes here:
![image](https://user-images.githubusercontent.com/6338661/52799178-3cbd7e00-3071-11e9-8474-49af55cf952c.png)

![image](https://user-images.githubusercontent.com/6338661/52799122-29121780-3071-11e9-8835-7099d366f158.png)